File: lab3.java
public class lab3 {
    public static void lab3(String[] args) {
        //9212
        //C3=2, Тип текстових змінних -> String
        //C17=15, Дія з рядком -> В заданому тексті замінити слова заданої довжини визначеним рядком
    	
	    Scanner in = new Scanner(System.in);
	    System.out.print("Input sentence: ");
	    String sentence = in.nextLine();
	    System.out.print("Input desired word: ");
	    String word = in.nextLine();
	    System.out.print("Input number of letters: ");
	    int number = in.nextInt();
	    String result = "";

	    String[] words = sentence.split(" ");
	    for (int i = 0; i < words.length; i++){
	      if (words[i].length() == number) {
	        words[i] = word;
	          }
	        }
	    for (int j =0; j < words.length; j++){
	      result += words[j] + " ";
	        }
	    System.out.println(result);
      
      
     
    }
}

File: lab3.py
# 9212
# C3=2, Тип текстових змінних -> String
# C17=15, Дія з рядком -> В заданому тексті замінити слова заданої довжини визначеним рядком

class Main:
    def __init__(self, sentence, number, word):
        self.number = number
        self.sentence = sentence
        self.word = word
        self.result = ''

    def main(self):
        s = self.sentence
        n = self.number
        w = self.word
        words = s.split(" ")
        print(words)
        for i in range(len(words)):
            if len(words[i]) == n:
                words[i] = w
        for j in range(len(words)):
            self.result += words[j] + " "
        return self.result

    def start(self):
        print(self.main())


se = input('Введіть текст :')
numb = input('Введіть число знаків :')
wor = input('Бажане слово:')
nio = Main(sentence=se, number=int(numb), word=wor)
nio.start()



